packages/firebase_messaging/lib/firebase_messaging.dart

```diff
@@ -3,13 +3,41 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
+import 'dart:ui';
 
 import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
 import 'package:meta/meta.dart';
 import 'package:platform/platform.dart';
 
 typedef Future<dynamic> MessageHandler(Map<String, dynamic> message);
 
+const String _backgroundName = 'plugins.flutter.io/android_fcm_background';
+
+void _fcmSetupCallback() {
```

Comment:
You might want to put this code in the one place it's used, or in a static method of the class, it's unusual to have it in a function at the global scope.

Reply:
Added as a static member of FirebaseMessaging